CONTRIBUTING.md

```diff
@@ -6,6 +6,14 @@ The team behind the [Artifact Wagon](https://github.com/heremaps/here-artifact-m
 accepts contributions via [pull requests](https://help.github.com/articles/about-pull-requests/) filed
 against the [GitHub project](https://github.com/heremaps/here-artifact-maven-wagon/pulls).
 
+## System requirements
+* Operating system:
+    * Windows 10
+    * MAC OSX 10
+    * Linux (Ubuntu 16.04)
```

Comment:
What about a higher version of Ubuntu?

Reply:
Note added to clarify usage of more fresh versions